src/dbus_api/api/manager_2_0/methods.rs

```diff
@@ -100,7 +101,11 @@ pub fn destroy_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
         }
     };
 
-    let msg = match dbus_context.engine.borrow_mut().destroy_pool(pool_uuid) {
+    let mut engine = dbus_context.engine.borrow_mut();
+
+    let result = pre_pool_op! {"preparing to destroy pool with uuid {}", pool_uuid; "successfully destroyed pool with uuid {}"; engine.destroy_pool(pool_uuid)};
```

Comment:
Whenever you pass in a UUID to a format string, you must add `.to_simple_ref()` at the end to remove the hyphens.

Reply:
Ah, okay; ".to_simple_ref()" is what I'm looking for (I knew it was ".to_simple...", but didn't know the exact string).  However, I can't seem to simply add it in that line, because the compiler seems to think the "." character is a token, and is unexpected.

Reply:
We discussed this and Bryan is going to change from `ident` fragment specifiers to `expr` fragment specifiers. This was my mistake; it probably should have been this from the beginning.

Reply:
Resolved.